astropy/time/formats.py

```diff
@@ -1103,16 +1111,33 @@ def to_value(self, timezone=None, parent=None, out_subfmt=None):
 
         for iy, im, id, ihr, imin, isec, ifracsec, out in iterator:
             if isec >= 60:
-                raise ValueError(
-                    f"Time {(iy, im, id, ihr, imin, isec, ifracsec)} is within "
-                    "a leap second but datetime does not support leap seconds"
-                )
+                isec = isec - 1
+                leap_second = True
+            else:
+                leap_second = False
+
             if timezone is not None:
-                out[...] = datetime.datetime(
+                dt = datetime.datetime(
                     iy, im, id, ihr, imin, isec, ifracsec, tzinfo=TimezoneInfo()
                 ).astimezone(timezone)
             else:
-                out[...] = datetime.datetime(iy, im, id, ihr, imin, isec, ifracsec)
+                dt = datetime.datetime(iy, im, id, ihr, imin, isec, ifracsec)
+
+            if leap_second:
+                dt += datetime.timedelta(seconds=1)
```

Comment:
This is a choice, maybe the right one, but it should be documented...

Reply:
Done via example.